test/e2e/metamask-ui.spec.js

```diff
@@ -772,9 +772,11 @@ describe('MetaMask', function () {
       await driver.delay(regularDelayMs * 2)
 
       await driver.clickElement(By.xpath(`//button[contains(text(), 'Create Token')]`))
-      await driver.delay(largeDelayMs)
+      await driver.wait(async () => {
+        windowHandles = await driver.getAllWindowHandles()
+        return windowHandles.length > 2
+      }, 10000)
```

Comment:
Nit: we have a method for this already apparently! I had forgotten about it until I went looking
```suggestion
      await waitUntilXWindowHandles(2)
```

Curious implementation though :thinking:  I'm not sure why it waits in "steps" - it seems needlessly complicated and slow.